src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,31 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.util.List;
+
```

Comment:
Avoid using wildcard imports to ensure that only necessary classes are imported. Configure your IDE to disable wildcard imports.